src/js/services/lang.js

```diff
@@ -45,7 +47,7 @@ const setLanguageSelectorOnLoad = () => {
     languages[window.currentLang] ||
     languages[window.navigator.language] ||
     languages['en'];
-  $(currentLang.domEl).addClass('selected');
+  $(currentLang.activeLang).toggleClass('no-display', false);
```

Comment:
I think this only works because we are actually changing the url to change the language. I think it works but just something to keep in mind.